docs/source/conf.py

```diff
@@ -14,7 +14,7 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath("../"))
+sys.path.insert(0, os.path.abspath("../../"))
```

Comment:
NIT: This is not platform independent as written. Probably not going to be running this on Windows, but `os.path.join("..", "..")` would be cleaner

Reply:
Probably Linux only but a good point all the same. Updated to be platform agnostic.